Assets/MixedRealityToolkit/_Core/Interfaces/IMixedRealityController.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
+{
+    /// <summary>
+    /// Mixed Reality Toolkit controller definition, used to manage a specific controller type
+    /// </summary>
+    public interface IMixedRealityController
+    {
+        /// <summary>
+        /// Outputs the current state of the Input Source, whether it is tracked or not. As defined by the SDK / Unity.
+        /// </summary>
+        ControllerState ControllerState { get; }
+
+        /// <summary>
+        /// The designated hand that the Input Source is managing, as defined by the SDK / Unity.
+        /// </summary>
+        Handedness ControllerHandedness { get; }
+
+        /// <summary>
+        /// The registered Input Source for this controller
+        /// </summary>
+        IMixedRealityInputSource InputSource { get; }
+
+        /// <summary>
+        /// Mapping definition for this controller, linking the Physical inputs to logical Input System Actions
+        /// </summary>
+        Dictionary<DeviceInputType, InteractionDefinition> Interactions { get; }
```

Comment:
Still not sure this should use the device input type as the key

Reply:
It's the simplest way to provide 1-1 mapping with a physical input without lengthy lookups or other costly mechanisms.  But agree that we should try and iterate on this later, for now, this works for the MVP.